engine/src/main/java/org/entando/entando/aps/system/services/pagemodel/PageModelService.java

```diff
@@ -52,7 +52,7 @@ public class PageModelService implements IPageModelService, ApplicationContextAw
 
     private final EntLogger logger = EntLogFactory.getSanitizedLogger(getClass());
     
-    public static final String TYPE_PAGE_MODEL = "pageModel";
+    public static final String TYPE_PAGE_MODEL = ComponentUsageEntity.TYPE_PAGE_TEMPLATE;
```

Comment:
why another constant with the same value? if you really intend to fix things in the future you should use `TYPE_PAGE_TEMPLATE` directly

Reply:
Done